docs/readme.md

```diff
@@ -75,6 +75,10 @@ Client may be generated manually using `testing_v1.json` and the master branch o
 
 - `gradle build`
 
+## Analytics
+
+Flank makes use of various analytics. For more information on what they are and how to control them please see [analyitics.md](analytics.md)
```

Comment:
I think that we should be consistent here, later on, we said that only Bugsnag is used (not various analytics)
```suggestion
Flank makes use of Bugsnag for analytics. For more information about analytics and how to control them please see [analyitics.md](analytics.md)
```

What do you think?

Reply:
Agreed. Thanks @piotradamczyk5. I thought it used various analytics at first beyond Bugsnag. But @bootstraponline cleared that it was only Bugsnag and i did not clean this definition up